spec/controllers/admin/admin_users_controller_spec.rb

```diff
@@ -439,38 +439,57 @@ def expect_changes_made_by(admin, log_items)
 
   describe "POST #destroy_user_creations" do
     let(:admin) { create(:admin) }
-    let(:user) { create(:user, banned: true) }
+    let(:user) { create(:user) }
+    let!(:work) { create(:work, authors: [user.default_pseud]) }
+    let(:other_owner) { create(:user, banned: false) }
+    let!(:collection1) { create(:collection) }
+    let!(:collection2) { create(:collection) }
+
+    before do
+      # Banning user only after creating works for them
+      user.update(banned: true)
```

Comment:
We don't have the cop enabled for it right now, but particularly in tests it's better to use `update!` to avoid silent failures: https://github.com/rubocop/rubocop-rspec/issues/911
```suggestion
      user.update!(banned: true)
```

Reply:
Updated.